### PR TITLE
AWP-1638: Add Actions Dropdown

### PR DESCRIPTION
**Pull request by ehsan-atif**
[AWP-1638](https://team-1637335847501.atlassian.net/browse/AWP-1638)
- add actions for more  options in dropdown

[AWP-1638]: https://team-1637335847501.atlassian.net/browse/AWP-1638?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -28,6 +29,7 @@ const Dropdown = <T,>({
   onSelect,
   renderOption = (option: T | null) => option?.toString(),
   label,
+  dropdownIcon = true,
```
Comment (darnold001) [inline src/components/dropdown/Dropdown.tsx]: Let's always try and default to false and have the action be true.

**Review by darnold001 — APPROVED** [review]
Looks good, just a minor comment. It also looks like this now has a dependency on the svg work you did.

**Review by zamanaf — APPROVED** [review]
LGTM
I think we are missing an export from index.ts

**Comment by zamanaf** [comment]
image
@ehsan-atif can you also update the styles as well they are a bit off

**Comment by ehsan-atif** [comment]
> LGTM I think we are missing an export from index.ts
 we already have component export in index

**Pull request merged**